src/glabs-left/web/index.scss

```diff
@@ -0,0 +1,95 @@
+@import '_core';
+
+a {
+  text-decoration: none;
+}
+
+h1, p {
+  margin: 0;
+}
+
+.container {
+  font-family: "Guardian Text Sans Web","Helvetica Neue",Helvetica,Arial,"Lucida Grande",sans-serif;
+  background-color: $neutral-8;
+  border-top: 2px solid $paid-article-brand;
+
+  display: flex;
+  flex-direction: column;
+
+
+  width: $gs-gutter * 6.5;
+
+  @include mq(tablet) {
+    width: $gs-gutter * 11;
+  }
+}
+
+.margin {
+  margin: 5px;
+}
+
+.body {
+  flex-grow: 1;
+  flex-shrink: 1;
+  flex-basis: auto;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+.thumbnail {
+  max-height: 40%;
+  overflow:hidden;
+}
+
+.thumbnail__img {
+  width: 100%;
+}
+
+.body__title {
+  font-size: 10px;
+  font-weight: bold;
+  color: $neutral-1;
+
+  @include mq(tablet) {
+    font-size: 12px;
+  }
+
+  margin-bottom: 5px;
+}
+
+.body__standfirst {
+
+  display: none;
+
+  color: $neutral-1;
+  font-size: 10px;
+
+  @include mq(tablet) {
+    font-size: 12px;
+    display: block;
+  }
+
+  margin-bottom: 5px;
+}
+
+.body__tagline {
+
+  color: $neutral-2;
+  font-size: 9px;
+
+  @include mq(tablet) {
+    font-size: 10px;
+  }
+
+  margin-top: 4px;
+  margin-left: auto;
+  text-align: right;
```

Comment:
While I'm nitpicking (sorry!), it would be good to consolidate the styles (so they are not split by the `mq`) and strip out the whitespace. For example, `body__tagline` becomes:

```
.body__tagline {
  color: $neutral-2;
  font-size: 9px;
  margin-top: 4px;
  margin-left: auto;
  text-align: right;

  @include mq(tablet) {
    font-size: 10px;
  }
}
```